pandas/tests/io/parser/test_parse_dates.py

```diff
@@ -1516,3 +1516,14 @@ def test_hypothesis_delimited_date(date_format, dayfirst, delimiter, test_dateti
 
     assert except_out_dateutil == except_in_dateutil
     assert result == expected
+
+
+def test_missing_column(all_parsers):
+    parser = all_parsers
+    content = StringIO("time,val\n" "212.23, 32\n")
```

Comment:
I believe that the CI will fail because, this string is not concatenated, can ou please concatenate them?